docs/Using-Tensorboard.md

```diff
@@ -21,10 +21,8 @@ session running on port 6006 a new session can be launched on an open port using
 the --port option.
 
 **Note:** If you don't assign a `run-id` identifier, `mlagents-learn` uses the
-default string, "ppo". All the statistics will be saved to the same sub-folder
```

Comment:
This was also missed in the release docs - the new behavior is that it will delete old summaries if run with `--force`